src/components/oButton.js

```diff
@@ -0,0 +1,108 @@
+import o from "ojs-core";
+import oInput from "./oInput";
+import oTextarea from "./oTextarea";
+
+//NOTE: config jak teraz czy jednak tak jak w inpucie?
+export default function oButton(config) {
```

Comment:
Sam typ jakbyśmy mogli podawać też spoko. Ale ile masz tych typów, chyba tylko 3: button | submit | reset
To jak już uważasz. Defaultowo dałbym zawsze type="button" to ważne bo pierwszy button w fomularzu nawet jak nie ma type="submit" to będzie automatycznie type "submit" i może to napsuć o ile dobrze pamiętam

Reply:
tak też będzie Panie Macieju. Defaultowo będzie type=button.